### PR TITLE
handle posts without districts

### PR DESCRIPTION
**Pull request by fgregg**
This PR allows for the resolution of posts that do not have districts. Sorry for the slow slicing.

### CODE REVIEW

File: pupa/importers/memberships.py [hunk 1]
```diff
@@ -43,7 +43,8 @@ def prepare_for_db(self, data):
 
         data['organization_id'] = self.org_importer.resolve_json_id(data['organization_id'])
         data['person_id'] = self.person_importer.resolve_json_id(data['person_id'])
-        data['post_id'] = self.post_importer.resolve_json_id(data['post_id'])
+        data['post_id'] = self.post_importer.resolve_json_id(data['post_id'],
+                                                             allow_no_match=True)
```
Comment (jpmckinney) [inline pupa/importers/memberships.py]: `allow_no_match` is never used?

**Comment by jpmckinney** [comment]
Nevermind prior comment (deleted). LGTM.

**Comment by jpmckinney** [comment]
Hmm, actually, I think I have an issue with `allow_no_match=True`. I sometimes get useful errors when a post fails to be resolved, because the district name in the source HTML differs from the district name that I loaded earlier from OCD divisions.

I think `allow_no_match=True` was only added here because of the change to the `else` branch in `add_term`. Why not add another parameter to `add_term` instead, to indicate "please create a district-less post"?

**Comment by jpmckinney** [comment]
Also, let's start editing the CHANGELOG in every PR.

**Comment by fgregg** [comment]
While there's all this activity, can we merge this? @jamesturk

**Comment by jpmckinney** [comment]
@fgregg I had some new comments though ^

**Comment by fgregg** [comment]
That makes sense to me @jpmckinney. Is this the approach you want?

**Comment by jpmckinney** [comment]
Yup. Looks good. Can you add a line to the changelog?

**Comment by fgregg** [comment]
done!

**Comment by jpmckinney** [comment]
Hmm, weird, where did the other commits go?

**Comment by fgregg** [comment]
ugh... weird.

**Comment by fgregg** [comment]
some rebase weirdness, changes are here again.

**Comment by fgregg** [comment]
thank you.

**Pull request merged**